app/controllers/application_templates_controller.rb

```diff
@@ -19,6 +19,11 @@ def new
   def show
   end
 
+  def toggle_active
+    @template.toggle!(:active)
```

Comment:
Oh, interesting. That's different from the usual Rails convention for what a bang is - I would've expected an exception instead of a silent failure (à la `update` vs. `update!`). I agree with your thinking.